core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -266,9 +266,10 @@ class RequestSendThread(val controllerId: Int,
 
         val response = clientResponse.responseBody
 
-        stateChangeLogger.withControllerEpoch(controllerContext.epoch).trace(s"Received response " +
-          s"${response.toString(requestHeader.apiVersion)} for request $api with correlation id " +
-          s"${requestHeader.correlationId} sent to broker $brokerNode")
+        if (stateChangeLogger.isTraceEnabled)
+          stateChangeLogger.withControllerEpoch(controllerContext.epoch).trace(s"Received response " +
+            s"${response.toString(requestHeader.apiVersion)} for request $api with correlation id " +
+            s"${requestHeader.correlationId} sent to broker $brokerNode")
```

Comment:
Why is this trace?

Reply:
There was an earlier comment on the PR suggesting we were more selective with what we log at INFO. I had considered reverting it from INFO here (https://github.com/apache/kafka/pull/6878#issuecomment-501214135) and received no comments against it, so I thought it was OK.

This isn't a per-partition log though, so I agree we don't need to keep it at trace